packages/recomponents/src/components/r-input/r-input.vue

```diff
@@ -226,10 +241,10 @@
                 this.$emit('key-down', event);
             },
             getFocus() {
-                this.$el.querySelector('input').focus();
+                this.$el.querySelector('input,textarea').focus();
             },
             blur() {
-                this.$el.querySelector('input').blur();
```

Comment:
How about to use $ref instead of querySelector here?

Reply:
I also prefer to use $refs here: https://github.com/Rebilly/rebilly-recomponents/pull/54/commits/222f949c1e3dd6429e545978b2dc9074ada99dec